### PR TITLE
feat: Implement a feature flag to hide Schedule and Filter options - EXO-71131 - Meeds-io/MIPs#119

### PR DESCRIPTION
**Pull request by sofyenne**

### CODE REVIEW

File: content-webapp/src/main/webapp/news/components/NewsApp.vue [hunk 1]
```diff
@@ -202,28 +215,32 @@ export default {
...
-      }
-      if (searchQueryParam) {
-        // set search value
-        this.searchText = searchQueryParam;
-      }
-      if (spacesFilterParam) {
-        // set search value
-        this.spacesFilter = spacesFilterParam;
+    this.$featureService.isFeatureEnabled('NewsScheduleAndFilterDisplaying').then(enabled => {
+      this.NewsScheduleAndFilterDisplaying = enabled;
```
Comment (azayati) [inline content-webapp/src/main/webapp/news/components/NewsApp.vue]:
```suggestion
this.$featureService.isFeatureEnabled('NewsScheduleAndFilterDisplaying').then(enabled => this.NewsScheduleAndFilterDisplaying = enabled);
```

File: content-webapp/src/main/webapp/news/components/NewsApp.vue [hunk 2]
```diff
@@ -202,28 +215,32 @@ export default {
...
+          // set search value
+          this.searchText = searchQueryParam;
+        }
+        if (spacesFilterParam) {
+          // set search value
+          this.spacesFilter = spacesFilterParam;
+        }
+
+      } else if (filterQueryParam === null) {
+        this.newsFilter = enabled ? 'all' : this.filterOptions[0].value;
```
Comment (azayati) [inline content-webapp/src/main/webapp/news/components/NewsApp.vue]:
```suggestion
        this.newsFilter = this.filterOptions[0].value;
```

**Review by azayati — CHANGES_REQUESTED** [review]

File: content-webapp/src/main/webapp/news/components/NewsApp.vue (hunk 1 above)
Comment (azayati) [reply content-webapp/src/main/webapp/news/components/NewsApp.vue]: IMo you can consider this modification https://github.com/Meeds-io/content/pull/27/files#r1576658694
The next block is independent from setting this.NewsScheduleAndFilterDisplaying

File: content-webapp/src/main/webapp/news/components/NewsApp.vue [hunk 3]
```diff
@@ -141,9 +138,25 @@ export default {
         month: 'long',
         day: 'numeric',
       },
+      NewsScheduleAndFilterDisplaying: false,
```
Comment (azayati) [inline content-webapp/src/main/webapp/news/components/NewsApp.vue]:
```suggestion
      newsScheduleAndFilterDisplaying: false,
```

File: content-webapp/src/main/webapp/schedule-news-drawer/components/ExoScheduleNewsDrawer.vue [hunk 4]
```diff
@@ -302,6 +303,7 @@ export default {
     audience: null,
     selectedAudience: null,
     disabled: true,
+    NewsScheduleAndFilterDisplaying: false,
```
Comment (azayati) [inline content-webapp/src/main/webapp/schedule-news-drawer/components/ExoScheduleNewsDrawer.vue]:
```suggestion
    newsScheduleAndFilterDisplaying: false,
```

**Review by azayati — CHANGES_REQUESTED** [review]

File: content-webapp/src/main/webapp/news/components/NewsApp.vue (hunk 1 above)
Comment (sofyenne) [reply content-webapp/src/main/webapp/news/components/NewsApp.vue]: This should be executed inside the "then" method after initializing the "NewsScheduleAndFilterDisplaying"  to correctly compute the filter options !

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**